src/main/webapp/app/lecture/shared/lecture-unit-fullscreen-layout/lecture-unit-fullscreen-layout.component.scss

```
@@ -85,58 +97,92 @@
         }
     }
+}
+
+// PrimeNG p-splitter styling: reproduce the previous split.js look (transparent gutter
+// with two small grip bars) and let panels fill the fullscreen container.
+:host ::ng-deep .fullscreen-splitter {
+    &.p-splitter {
+        flex: 1;
+        min-width: 0;
+        min-height: 0;
+        // Strip PrimeNG's default chrome so the splitter blends into the fullscreen surface (matches the old split.js layout).
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)